Solutions/BitSight/Parsers/BitSightAlerts.txt

```diff
@@ -0,0 +1,42 @@
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias (e.g. BitSightAlerts).
+// Function usually takes 10-15 minutes to activate. You can then use function alias from any other queries (e.g. BitSightAlerts | take 10).
+// References : 
+// Using functions in Azure monitor log queries : https://docs.microsoft.com/azure/azure-monitor/log-query/functions
+// Tech Community Blog on KQL Functions : https://techcommunity.microsoft.com/t5/Azure-Sentinel/Using-KQL-functions-to-speed-up-analysis-in-Azure-Sentinel/ba-p/712381
+//
+let Alerts_view  = view () { 
+    Alerts_data_CL
+    | extend 
+                EventVendor ="AlertBitSight",
```

Comment:
Vendor should be BitSight and Product should be Alert.
This is applicable to all the Parsers.

Reply:
@devikamehra Changes done as above.